engine/src/main/java/org/hibernate/search/query/dsl/QueryBuilder.java

```diff
@@ -44,6 +44,13 @@
 	PhraseContext phrase();
 
 	/**
+	 * Build a plain text query.
```

Comment:
Should probably be "Build a query from a simple query string"... ? Same for the `@return` below.